frontend/app/src/locales/en.json

```diff
@@ -665,12 +665,12 @@
     "session": {
       "add_queriable_address": {
         "error": {
-          "message": "Failure to add a queriable address: {message}"
```

Comment:
Here and below there is no reason to change this. queriable is a valid word: https://en.wiktionary.org/wiki/queriable